README.md

```diff
@@ -86,6 +86,14 @@ markers are roughly level with the observer:
 
 Available as a typed named tuple `Spherical` from `Token.spherical`.
 
+### Orientation
+
+Rotation of the marker around its centre (relative to the observer)
+ - `rot_x`: rotation about the Cartesian x-axis, positive angles represent the front going upwards.
```

Comment:
This differs from what's noted in `robot-api`. Is that intentional?

Reply:
good catch, fixed in 638580b5db8898239c186aba8ba2ad0bf8f7cfef